theano/tensor/signal/pool.py

```diff
@@ -645,7 +645,7 @@ def infer_shape(self, node, in_shapes):
 
 class MaxPoolGrad(PoolGrad):
     def __init__(self, ds, ignore_border, st=None, padding=(0, 0)):
```

Comment:
I would add the openmp parameter to this method to allow the user to change it as it is for the other openmp op.